website/www/site/data/en/committer_traits.yaml

```diff
@@ -0,0 +1,21 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- body: { text: To learn more see the ASF documentation., link: https://www.apache.org/foundation/policies/conduct.html }
+  bolded: Knows, upholds, and reinforces the Apache Software Foundation code of conduct.
+  icon: icons/contributor/become a committer/messages-icon.svg
+- body: { text: To learn more see the ASF documentation., link: https://infra.apache.org/committers.html#committer-responsibilities }
+  bolded: Knows, upholds, and reinforces the responsibilities of an Apache Software Foundation committer.
+  icon: icons/contributor/become a committer/beam-logo-icon.svg
+- body: { text: To learn more see our best practices!, link: https://beam.apache.org/contribute/become-a-committer/ }
```

Comment:
become-a-committer is the page that this is displayed on. I think this link should be /attributes/?

This seems a little backwards again.. but that is a design question for @griscz

Reply:
```suggestion
- body: { text: To learn more see our best practices!, link: https://beam.apache.org/contribute/attributes/ }
```